README.rst

```diff
@@ -67,10 +67,11 @@ please `read the contribution guidelines`_.
 License
 =======
 
-``hyper-h2`` is made available under the MIT License. For more details, see the
+``h2`` is made available under the MIT License. For more details, see the
 ``LICENSE`` file in the repository.
 
 Authors
 =======
 
-``hyper-h2`` is maintained by Cory Benfield, with contributions from others.
+``h2`` was authored by Cory Benfield and is maintained
```

Comment:
Changed this section to more accurately reflect the maintainership of the package